app/models/claim.rb

```diff
@@ -33,8 +33,15 @@ class Claim < ApplicationRecord
   has_many :mentor_trainings
   has_many :mentors, through: :mentor_trainings
 
+  validates :status, presence: true
   validates :reference, uniqueness: { case_sensitive: false }, allow_nil: true
 
+  scope :not_internal, -> { where(status: %w[draft submitted]) }
```

Comment:
```suggestion
  scope :not_internal, -> { where.not(status: :internal) }
```

Could we use `where.not(...)` here?